generators/server/templates/src/main/docker/app.yml.ejs

```diff
@@ -64,7 +64,7 @@ _%>
       # - JHIPSTER_CACHE_REDIS_CLUSTER=true
 <%_ } _%>
 <%_ if (authenticationTypeOauth2) { _%>
-      - SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_OIDC_ISSUER_URI=http://keycloak:9080/auth/realms/jhipster
+      - SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_OIDC_ISSUER_URI=http://keycloak:8080/realms/jhipster
```

Comment:
Should it be 9080 as thats the exposed port?

Reply:
I think the exposed port is keycloak:8080 but it is exposed to localhost:9080

Reply:
@atomfrede by the way half the tests work so I am pretty sure this is correct

Reply:
Yes guess you're right. Just wanted to make sure as it was 9080 before, looked strange.

Reply:
this should be `9080` as that's exposed for outside services to connect

Reply:
@vishal423 did you try it ? It was 9080 before, and I changed it because of how docker exposes ports.

Reply:
are you saying there is a recent change in `docker` ports mapping behaviour? If not, then, your configuration is incorrect. The previous configuration used to work and was thoroughly tested in local and CI (with docker compose).